FlowCrypt/Functionality/Services/Key Services/KeyService.swift

```diff
@@ -68,51 +68,16 @@ final class KeyService: KeyServiceType {
 
```

Comment:
Please change this method signature to not accept any pass phrase. It should just itself return [PrvKeyInfo] with pass phrases optionally included (nil when not available for that key). Based on all pass phrases that are available in storage OR in memory.